src/_guides/language/type-system.md

```diff
@@ -457,9 +457,10 @@ What about going in the other direction?
 Can you assign an `Animal` list to a `List<Cat>`?
 
 {:.fails-sa}
-<?code-excerpt "lib/strong_analysis.dart (generic-type-assignment-Animal)" replace="/Animal/[!$&!]/g"?>
+<?code-excerpt "lib/strong_analysis.dart (generic-type-assignment-Animal)" replace="/<Animal/<[!Animal!]/g"?>
 {% prettify dart tag=pre+code %}
-List<Cat> myCats = <[!Animal!]>[];
+List<[!Animal!]> myAnimals = ...
```

Comment:
Maybe not following the example, but should you add a similar declaration for the `MaineCoon` list above line 453? Does that help with clarity also?